src/components/Users.js

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useState } from "react";
+import { BaseUrl } from "../Constants";
+import axios from "axios";
+import localStorageService from "../LocalStorageService";
+import Table from "react-bootstrap/Table";
+import { Spinner } from "react-bootstrap";
+
+export default function Users() {
+  const [users, setUsers] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const token = localStorageService.get("auth-token");
+    let config = {
+      method: "get",
+      maxBodyLength: Infinity,
+      url: BaseUrl + "chat/users/",
+      headers: {
+        Authorization: `token ${token}`,
+      },
+    };
+
+    axios
+      .request(config)
+      .then((response) => {
+        setIsLoading(false);
+        setUsers(response.data);
+      })
+      .catch((error) => {
+        console.log(error);
```

Comment:
Similarly if we have any error, we need to set the loading to false as well so use `setIsLoading(false)` here too.